src/server/images/models.js

```diff
@@ -94,8 +71,38 @@ export default class ImageManager {
   */
   static getImageObjList(imageKeys) {
     const s3 = new S3Connector();
-    return imageKeys.map((imageKey)=>{
+    return imageKeys.map((imageKey)=> {
       return {imageKey, imageUrl: s3.getImageUrl(imageKey)};
     });
   }
+
+ /**
+  * @getImageKeys
+  * get sorted image keys using item key.
+  * @param {string} itemKey imageKey
+  * @param {Array} checkState checkState
+  * @return {Array} image keys
+  */
+  static getImageKeys(itemKey, checkState = [STATE.ALIVE, STATE.EXPIRED]) {
+    const keys = [];
+    return Promise.all(checkState.map((state) => {
+      return new Promise((resolve, reject) => {
```

Comment:
I indented to sort after Promise.all, so return Promise object cause fetchPrefix function's return value is not Promise object.